service/frontend/errors.go

```diff
@@ -35,6 +35,7 @@ var (
 	errExecutionNotSet                                    = serviceerror.NewInvalidArgument("Execution is not set on request.")
 	errWorkflowIDNotSet                                   = serviceerror.NewInvalidArgument("WorkflowId is not set on request.")
 	errActivityIDNotSet                                   = serviceerror.NewInvalidArgument("ActivityId is not set on request.")
+	errActivityIdOrTypeNotSet                             = serviceerror.NewInvalidArgument("Either activity ID or activity type should be set on request.")
```

Comment:
I stopped fighting in `Id` vs `ID` war but here, inconsistency is so obvious.
```suggestion
	errActivityIDOrTypeNotSet                             = serviceerror.NewInvalidArgument("Either Activity.Id or Activity.Type should be set on request.")
```

Reply:
changed. (for some reason don't see it in IDE)